### ISSUE TITLE
unknown or uninitialised column when using `forestly::format_ae_listing()`

### ISSUE THREAD

**Issue by yuliasidi** [Merck/forestly#17]
The below code causes a warning message of "unknown or uninitialised column" due to the fact that `res$Outcome` is not initially defined in the `res` data. Please note that this issue is present when the `dataset_adae` is tibble and not data.frame. Since in practice adae dataset could be generated by SAS and read into R using `haven::read_sas()`, i.e., it will be by default a tibble, please consider initializing `res$Outcome` and `res$res$Action_Taken` in the `forestly::format_ae_listing()` to avoid a warning message.

```r
library(forestly)

forestly::meta_forestly(
  dataset_adsl = forestly_adsl,
  dataset_adae = forestly_adae |> tibble::as_tibble(),
  population_term = "apat",
  population_subset = SAFFL == "Y",
  observation_term = "wk12",
) |>
  forestly::prepare_ae_forestly(parameter = "any;rel",
                      ae_listing_display = c("SITEID","AGE", "SEX", "RACE", "AEOUT"))
```

**Comment by elong0527** [comment]
Thanks for reporting the issue. It should be fixed by the PR above.

There are deeper problem in `ae_listing.R`, documented in #23

**Comment by nanxstats** [comment]
Fixed by #22
